web/styles/widgets.css

```diff
@@ -106,7 +106,6 @@
 .todo-widget,
 .poll-widget {
     & h4 {
-        font-size: 18px;
```

Comment:
I think we should put an explicit `font-size: 1.1em`  here, and/or make this reference a CSS class. The `h4` bootstrap logic is something we intend to delete as soon as we get around to doing a sweep, so it's best to not avoid additional logic depending on it.

Reply:
Fixed [here](https://github.com/zulip/zulip/compare/57876ea827d502c775ce1088e5e66ed5b9b05f65..342158f67cbb82ac422eb1becb6dbeb2387b7770).

Screenshot for both poll and todo list at 20px font size after that change just for sanity check:
<img width="1040" alt="Screenshot 2025-01-22 at 2 19 56 AM" src="https://github.com/user-attachments/assets/1fb29397-2d1b-4e35-9c0a-33fcb9988d7e" />